README.md

```diff
@@ -115,11 +115,25 @@ Granular tasks exist for each source set in the project: `formatKotlin`*`SourceS
 
 ### Git Hooks
 
-Kotlinter can install a hook to run pre-push (`installKotlinterPrePushHook`). The hook runs `lintKotlin` and, if there are errors, `formatKotlin` and exits
-non-zero leaving changed files to be committed.
+Kotlinter can install a hook to run pre-push (`installKotlinterPrePushHook`). The hook runs `lintKotlin` and,
+if there are errors, `formatKotlin` and exits non-zero leaving changed files to be committed.
 
 You *must* apply the kotlinter plugin to your root project to make this task available.
 
+To install the hook automatically when someone runs the build, add this to your root project `build.gradle.kts`:
+
+```kotlin
+plugins {
+    id("org.jmailen.kotlinter") version "2.4.1" apply false
+}
+
+apply(plugin = "org.jmailen.kotlinter")
+
+tasks.check {
+    dependsOn("installKotlinterPrePushHook")
```

Comment:
I guess it could be noisy, but your other PR probably short circuits with no output if the hook is already installed?

Reply:
Correct, output is only generated if the hook is changed.